src/media/record/record_device.h

```diff
@@ -47,6 +47,9 @@ namespace librealsense
         void tag_profiles(stream_profiles profiles) const override { m_device->tag_profiles(profiles); }
         bool compress_while_record() const override { return true; }
         bool contradicts(const stream_profile_interface* a, const std::vector<stream_profile>& others) const override { return m_device->contradicts(a, others); }
+        double get_device_time() { return 0; }; // Returns time in miliseconds.
```

Comment:
It makes sense to incorporate units into the signature `get_device_time_ms() ` to avoid ambiguity